core/server/models/settings.js

```diff
@@ -81,7 +81,7 @@ Settings = ghostBookshelf.Model.extend({
         }).then(function () {
             var themeName = setting.value || '';
 
-            if (setting.key !== 'activeTheme') {
+            if (setting.key !== 'activeTheme' || options.importing) {
```

Comment:
Also this fix is for LTS-only IMO - what we do with alpha is a totally different question 👍